RequiredExports.h

```diff
@@ -52,6 +52,14 @@ struct AIModDesc
 	Export const char TechtreeName[] = techTreeName; \
 	Export const AIModDesc DescBlock = { missionType, numPlayers, maxTechLevel, bUnitOnlyMission, MapName, LevelDesc, TechtreeName, 0 };
 
+// Struct introduced in the official Sierra release update pack 1 (1.2.0.7)
+// Set aiPlayerCount to 1 to allow a computer controlled AI in a multiplayer scenario
+// Example: Export AIModDescEx aiModDescEx = { 1 };
+struct AIModDescEx {
+	int aiPlayerCount;

```

Comment:
I prefer this name to the num variant. Perhaps if it is represented this way elsewhere or for consistency it shouldn't be count though?

Reply:
We are trending towards "Count" in the OP2Utility based projects. I almost wrote it that way myself on the forums. Your point about consistency is valid though. I suspect the Outpost 2 API tends to use "num".

Actually, it gets worse. They have names like `TethysGame::NoPlayers()`. They're inconsistent too, also having `SetColorNumber(int colorIndex)`. And what do you know, they even have a few "counts", `TotalUnitCount()`, `UnitCount(UnitClassifactions unitType)`, `GetRLVCount()`.

Maybe this doesn't matter so much. It's already inconsistent, and we can't fix the built in names. At any rate, this is a fairly minor point, since it can be initialized without using the field name. Most people can probably get away without reading it by name either.

Maybe we should open an issue to address parameter and struct field naming. We do have the ability to change the names of parameters and struct fields without affecting linking.